test/maps/lua_testsuite.wmf/scripting/gettext.lua

```diff
@@ -6,10 +6,8 @@ test_descr = lunit.TestCase("Gettext Test")
 -- Code takes a different path when textdomain was pushed, so we have 2 variants of each test.
 
 function gettext_calls()
-   -- TODO(Nordfriese): Delete the syntax-only testcases when we enforce optional parentheses everywhere
-   assert_equal("foo", _"foo")
    assert_equal("foo", _("foo"))
-   assert_equal(_"Carrier", _("Carrier"))
+   assert_equal(_("bar"), _("bar"))
```

Comment:
this assert doesn't make sense. So either we delete this, or we keep the original code and exclude tst from the check
I vote for the latter

Reply:
This assert checks that the function is deterministic and always returns the same result when invoked with the same parameters. We have lots of checks for things that appear self-evident but strictly speaking aren't.

Since there is now no other place anywhere in the code that uses the notation without optional parentheses, coding an exception to the style checker just so we can check that a language syntax feature we choose not to use would work in principle, doesn't make sense IMHO.

Reply:
this would be checking the assert_equal funtion in a context to test gettext. This isn't intuitive and does not make sense imho. 
However as it doesn't hurt i have no hard feelings about this so your choice